### PR TITLE
Add new .text-break utility for applying word-break: break-word

### LINKED ISSUE #19315: Add utility class for word-wrap/word-break
Relevant pull request: https://github.com/twbs/bootstrap/pull/19316

Very long words can break out of its containing tag. 

One can force these long words to break to the next line(s) if necessary.

See an example on https://jsfiddle.net/AdrienBe/n16vqz5p/

Resources:
http://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/
http://stackoverflow.com/questions/320184/how-to-prevent-long-words-from-breaking-my-div
http://caniuse.com/#search=word-wrap
http://stackoverflow.com/a/22811590/759452
http://developer.mozilla.org/en-US/docs/Web/CSS/word-wrap

### PR DESCRIPTION
**Pull request by mdo**
If we need to be changing the breaking of content, we shouldn't be doing it piece meal on our components. A utility helps in the interim, hence this PR, but v5 will likely see this added to the `<body>` element in Reboot to solve the problem across the entire project. (At that point we'll likely need the opposite utility, too).

Fixes #19315.

### CODE REVIEW

File: scss/utilities/_text.scss [hunk 1]
```diff
@@ -62,6 +62,8 @@
 
 .text-decoration-none { text-decoration: none !important; }
 
+.text-break    { word-break: break-word !important; }
```
Comment (MartijnCuppens) [inline scss/utilities/_text.scss]: One space should do the job.
```suggestion
.text-break { word-break: break-word !important; }
```

**Comment by MartijnCuppens** [comment]
Looks good, maybe add some documentation?

**Comment by mdo** [comment]
@MartijnCuppens Added!

**Review by MartijnCuppens — APPROVED** [review]
Wonderful!

**Pull request merged**